src/App/App.js

```diff
@@ -1,29 +1,43 @@
 import './App.css';
-import { Routes, Route } from 'react-router-dom'
+import { Routes, Route, Navigate } from 'react-router-dom'
 import Favorites from '../Favorites/Favorites.js'
 import Form from '../Form/Form.js'
 import Home from '../Home/Home.js'
 import Nav from '../Nav/Nav.js'
 import Poster from '../Poster/Poster.js'
-import { mockQuotes, mockImages, mockPosters } from '../mockData'
+import Error from '../Error/Error.js'
 import {useState, useEffect } from 'react'
-import { getData } from '../apiCalls'
+import { getData, postFavorite } from '../apiCalls'
 
 const App = () => {
-  const [quotes, setQuotes] = useState(mockQuotes.quotes)
-  const [images, setImages] = useState(mockImages.images)
+  const [quotes, setQuotes] = useState([]);
+  const [images, setImages] = useState([]);
+  const [favorites, setFavorites] = useState([]);
+  const [poster, setPoster] = useState({});
+  const [fontSize, setFontSize] = useState('');
+  const [error, setError] = useState('');
+  const [isFavorite, setIsFavorite] = useState(false);
 
-  //// --> hopefully ready for real data - needs testing/debugging
-  // useEffect(() => {
-  //   getData(quotes)
-  //   .then(response => setQuotes(response.quotes))
-  //   .catch(error => alert(error.message))
+  useEffect(() => {
+    getData('quotes')
+    .then(response => setQuotes(response.quotes))
+    .catch(error => setError(error.message))
 
-  //   getData(images)
-  //   .then(response => setImages(response.images))
-  //   .catch(error => alert(error.message))
-  // }, [])
+    getData('images')
+    .then(response => setImages(response))
```

Comment:
interesting that this still works. I would imagine this line would have to be `.then(response => setImages(response.images))`